### ISSUE TITLE
Can't change Spot to dark theme

### ISSUE THREAD

**Issue by AllPurposeMat** [xou816/spot#388]
**Describe the bug**
Hi! I'm loving Spot so far, but ran into an issue. I've installed it using the AUR package and the app opens in light theme.
I have tried to toggle `prefers-dark-theme` using the dconf editor, but it seems to have no effect as the app still launches in light theme.

**To Reproduce**
1. Install spot-client from the AUR.
2. Launch the app (not as sudo).
3. Observe the app is in light theme instead of dark theme.

**Expected behavior**
The app should use the gorgeous dark theme displayed in the README.

**Screenshots**
![image](https://user-images.githubusercontent.com/55262273/143678063-7ad5667f-6696-431f-8afb-e2a3e49a799f.png)

**General information:**
 - Distribution: Arch Linux
 - Installation method: `yay -S spot-client` (from AUR)
 - Version: 0.2.2-1
 - Device used: Desktop

**Comment by TheCultLeader666** [comment]
The flatpak version uses the dark theme by default for me.

**Comment by xou816** [comment]
Hi! Thanks for the report.

This...
```
Using GtkSettings:gtk-application-prefer-dark-theme with libadwaita is unsupported. Please use AdwStyleManager:color-scheme instead.
```
...used to be a mere warning I think, but looks like now the old gtk setting does not work :( 

It's been fixed in the development version. I might merge dev into master soon, if I can't I'll just cherry pick the relevant change into master.

If you don't mind building it yourself, you can try out the development branch or fix the issue ([relevant lines](https://github.com/xou816/spot/blob/development/src/main.rs#L87)) and rebuild! :)

**Comment by xou816** [comment]
The flatpak version uses an older libadwaita version than what arch ships I think so it might be fine indeed!

**Comment by PJ-Singh-001** [comment]
I do not think this is a flatpak-only issue.

I built and installed Spot using [these instructions](https://github.com/xou816/spot/issues/513).

The gsettings key is set to use a dark theme:

    gsettings get dev.alextren.Spot prefers-dark-theme
    true

And this is reflected in the Spot Preferences dialog:

![image](https://user-images.githubusercontent.com/19394936/165843899-0477d02e-84a7-4ca1-aaab-1f0669fbba9f.png)

However, (as you can see from the screen clip of the Preferences dialog) Spot only uses the light theme.

This occurs on Ubuntu 22.04 in both Wayland and X11.

**Comment by xou816** [comment]
The code surrounding this has changed a bit since the original issue. There are probably more option that we have to expose besides light and dark, i think libadwaita lists 4 enum variants. Related to https://github.com/xou816/spot/issues/142#issuecomment-1099612688 as well

**Comment by xou816** [comment]
fixed recently #548
